### PR TITLE
Ubuntu/devel

### PR DESCRIPTION
**Pull request by holmanb**
## Proposed Commit Message

```
summary: no more than 70 characters

A description of what the change being made is and why it is being
made, if the summary line is insufficient.  The blank line above is
required. This should be wrapped at 72 characters, but otherwise has
no particular length requirements.

If you need to write multiple paragraphs, feel free.

LP: #NNNNNNN (replace with the appropriate bug reference or remove
this line entirely if there is no associated bug)
```

## Checklist:

 - [ ] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/contributing.html)
 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

**Comment by TheRealFalcon** [comment]
Since this is a hotfix, I think we can leave off the two commits (i.e., httpretty) since the last release.

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+ -- Brett Holman <brett.holman@canonical.com>  Fri, 30 Sep 2022 15:41:00 -0600
+
+cloud-init (22.3.3-0ubuntu1~22.10.1) kinetic; urgency=medium
+
+  * New upstream release.
+    + Release 22.3.3 (LP: #1986703)
+    + Fix Oracle DS not setting subnet when using IMDS (#1735)
+      (LP: #1989686)
+    + Release 22.3.2 (LP: #1986703)
```
Comment (blackboxsw) [inline debian/changelog:15]: For some reason we are seeing a diff here (and changelog items that were redacted on the previous upload to -proposed)
Have your started his brnach from what was previously released to -proposed. I expect we'd need to start from either `git checkout ubuntu/22.3.4-0ubuntu1_22.10.1` or git checkout upstream/ubuntu/kinetic-22.3-hotfix
```
 cloud-init (22.3.3-0ubuntu1~22.10.1) kinetic; urgency=medium
 
-  * New upstream release.
+  * New upstream bug fix release.
     + Release 22.3.3 (LP: #1986703)
     + Fix Oracle DS not setting subnet when using IMDS (#1735)
       (LP: #1989686)
-    + Release 22.3.2 (LP: #1986703)
     + azure: define new attribute for pre-22.3 pickles (#1725)
     + sources/azure: ensure instance id is always correct (#1727)
       [Chris Patterson]
-    + Release 22.3.1 (LP: #1986703)
 
  -- Brett Holman <brett.holman@canonical.com>  Mon, 19 Sep 2022 13:13:06 -0600
 
```

We don't want to see the deltas for LP: #1986703 in the changelog

**Review by blackboxsw — CHANGES_REQUESTED** [review]
Can you document in the PR description how you are drawing up this PR as I think you are starting from a branch/tag that doesn't align with what we previously released to -proposed.

**Review by blackboxsw — APPROVED** [review]
LGTM!
- only minor authorship or debian/changelog white-space/bug annotations
- changelog version is good for development release 22.3.4-0ubuntu1
-  content in changeset from 22.3.3 -> 22.3.4 is good
- no d/patches to apply/test
- build-package; sbuild no new lintian errors/warnings

We can merge into kinetic-22.3-hotfix branch and `git tag ubuntu/22.3.4-0ubuntu1`

**Comment by blackboxsw** [comment]
merged and pushed upstream tag `ubuntu/22.3.4-0ubuntu1`

**Pull request merged**